lib/ansible/modules/network/cnos/cnos_banner.py

```diff
@@ -0,0 +1,264 @@
+            variable C(ANSIBLE_NET_AUTHORIZE) will be used instead.
+        type: bool
+        default: 'no'
+      auth_pass:
+        description:
+          - Specifies the password to use if required to enter privileged mode
+            on the remote device.  If I(authorize) is false, then this argument
+            does nothing. If the value is not specified in the task, the value
+            of environment variable C(ANSIBLE_NET_AUTH_PASS) will be used
+            instead.
+"""
+
+EXAMPLES = """
+- name: configure the login banner
+  cnos_banner:
+    banner: login
+    text: |
+      this is my login banner
+      that contains a multiline
+      string
+    state: present
+
+- name: remove the motd banner
+  cnos_banner:
+    banner: motd
+    state: absent
+
+- name: Configure banner from file
+  cnos_banner:
+    banner:  motd
+    text: "{{ lookup('file', './config_partial/raw_banner.cfg') }}"
+    state: present
+
+"""
+
+RETURN = """
+commands:
+  description: The list of configuration mode commands to send to the device
+  returned: always
+  type: list
+  sample:
+    - banner login
+    - this is my login banner
+    - that contains a multiline
+    - string
+"""
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.connection import exec_command
+from ansible.module_utils.network.cnos.cnos import load_config, run_commands
+from ansible.module_utils.network.cnos.cnos import check_args
+from ansible.module_utils.network.cnos.cnos import cnos_argument_spec
+import re
+
+
+def map_obj_to_commands(updates, module):
+    commands = list()
+    want, have = updates
+    state = module.params['state']
+
+    if state == 'absent' and 'text' in have.keys() and have['text']:
+        commands.append('no banner %s' % module.params['banner'])
+
+    elif state == 'present':
+        if want['text'] and (want['text'] != have.get('text')):
+            banner_cmd = 'banner %s ' % module.params['banner']
+            for bline in want['text'].strip().splitlines():
+                final_cmd = banner_cmd + bline.strip()
+                commands.append(final_cmd)
+
+    return commands
+
+
+def map_config_to_obj(module):
+    rc, out, err = exec_command(module,
+                                'show banner %s' % module.params['banner'])
+    if rc == 0:
+        output = out
+    else:
+        rc, out, err = exec_command(module,
+                                    'show running-config | include banner %s'
+                                    % module.params['banner'])
+        if out:
+            output = re.search(r'\^C(.*)\^C', out, re.S).group(1).strip()
+        else:
+            output = None
+    obj = {'banner': module.params['banner'], 'state': 'absent'}
+    if output:
+        obj['text'] = output
+        obj['state'] = 'present'
+    return obj
+
+
+def map_params_to_obj(module):
+    text = module.params['text']
+    if text:
+        text = str(text).strip()
```

Comment:
```suggestion
        text = to_text(text).strip()
```
`to_text` can be imported from `ansible.module_utils._text`

Reply:
Review comments is taken care